Formula/ocaml-findlib.rb

```diff
@@ -31,7 +30,7 @@ def install
     system "./configure", "-bindir", bin,
                           "-mandir", man,
                           "-sitelib", HOMEBREW_PREFIX/"lib/ocaml",
-                          "-config", etc/"findlib.conf",
+                          "-config", pkgshare/"etc/findlib.conf", # TODO: DO NOT MERGE
```

Comment:
Needs fix for https://github.com/Homebrew/homebrew-test-bot/issues/805 to allow using `etc` files during CI build.